core/node/da_clients/src/eigen/sdk.rs

```diff
@@ -63,20 +77,23 @@ impl RawEigenClient {
             .await_for_inclusion(client_clone, disperse_reply)
             .await?;
 
-        let verification_proof = blob_info
-            .blob_verification_proof
-            .clone()
-            .ok_or_else(|| anyhow::anyhow!("No blob verification proof in response"))?;
+        let blob_info = blob_info::BlobInfo::try_from(blob_info)
+            .map_err(|e| anyhow::anyhow!("Failed to convert blob info: {}", e))?;
+        self.verifier
+            .verify_commitment(blob_info.blob_header.commitment.clone(), data)
```

Comment:
In this code, I was looking at [this file](https://github.com/lambdaclass/zksync-era/blob/main/core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs#L183). Where does eigenda fit in the `match (self.mode, self.pubdata_da)` result ?

Reply:
To clarify, is this computing the commitment locally to see if the DA has computed the same commitment ?

Reply:
We fit in `(L1BatchCommitmentMode::Validium, PubdataSendingMode::Custom)`
The `verifier.verifyCommitment` function is the one that generates a commitment based on the blob and  compares it to the one given by EigenDA.
The code you were looking at is the one that defines what would be sent to the L1, once we make the PR with full M0, that part will send the EigenDA commitment.